src/components/PostDetails.tsx

```diff
@@ -104,23 +90,23 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
       </div>
 
       <div className="block">
-        {!loaded && (
+        {loaded && (
           <Loader />
         )}
 
-        {loaded && hasError && (
+        {!loaded && hasError && (
           <div className="notification is-danger" data-cy="CommentsError">
             Something went wrong
           </div>
         )}
 
-        {loaded && !hasError && comments.length === 0 && (
+        {!loaded && !hasError && comments.length === 0 && (
           <p className="title is-4" data-cy="NoCommentsMessage">
             No comments yet
           </p>
         )}
 
-        {loaded && !hasError && comments.length > 0 && (
+        {!loaded && !hasError && comments.length > 0 && (
```

Comment:
```suggestion
        {!loaded && !hasError && !!comments.length && (
```

Reply:
i need rewrite api? seriosly? it wrote not me, this api be in react module, and those it is ok there is not?